features/channel/managing_channels/editing_channel.feature

```diff
@@ -33,3 +33,11 @@ Feature: Editing channel
     Scenario: Seeing disabled base currency field during channel edition
         When I want to modify a channel "Web Channel"
         Then the base currency field should be disabled
+
+    @ui @todo
+    Scenario: Changing type of channel
+        When I want to modify a channel "Web Channel"
+        And I define its type as "Website"
+        And I save my changes
+        Then I should be notified that it has been successfully edited
```

Comment:
```suggestion
        Then I should be notified that it has been successfully updated
```
?

Reply:
IMO, no, name of the feature is 'editing channel', and also in the previous step, we have 'edited'

Reply:
IMO it's irrelevant, `editing` is consistent with any other scenarios